includes/admin/payments/orders.php

```diff
@@ -684,6 +684,29 @@ function edd_order_details_overview( $order ) {
 		}
 	}
 
+	$has_tax  = 0;
```

Comment:
Lets make the default `false` to match the first condition [here](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/admin/views/tmpl-order-totals.php#L32)

Reply:
If I change it to `false`, it fails [this check](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/admin/views/tmpl-order-totals.php#L29) instead (I think `false` is just empty after localization and `0` is read as `false`, at least as far as I can tell?).

Reply:
Let's make it an explicit `false` string check so we don't have to figure out/guess what `wp_localize_script()` is doing.

Reply:
I'm okay with changing it but it does affect multiple templates (`order-totals`, `order-item`, `add-order-item` at least). Can I use `none`? I like that better than `no` for whatever reason.

Reply:
Yep, that works.